frontend/components/card/CardTopicSelection.vue

```diff
@@ -88,6 +101,36 @@ const moreOptionsShown = ref(false);
 const inputFocus = ref(false);
 const emit = defineEmits(["update:modelValue"]);
 
+const inputTabbing = (e: KeyboardEvent) => {
+  const firstTopic: HTMLElement | null = document.querySelector(".topic");
+
+  firstTopic?.focus();
```

Comment:
Is it possible to get the first topic being focussed even if we're tabbing back from the element after CardTopicSelection? As of now we're getting the last element, and I think consistency would be good,